subprojects/workers/src/integTest/groovy/org/gradle/workers/internal/WorkerExecutorServicesIntegrationTest.groovy

```diff
@@ -268,13 +273,18 @@ class WorkerExecutorServicesIntegrationTest extends AbstractWorkerExecutorIntegr
 
             task runInWorker(type: WorkerTask) {
                 isolationMode = $isolationMode
-                ext.testFile = project.file("\$buildDir/\$name")
+                def classpath = sourceSets.main.output.classesDirs
+                def projectDir = project.projectDir
+                def testFile = project.file("\$buildDir/\$name")
                 additionalParameters = {
-                    it.classpath.from(sourceSets.main.output.classesDirs)
+                    it.classpath.from(classpath)
                     it.setProjectDir(projectDir)
                     it.setTestFile(testFile)
                 }
-                doLast { assert testFile.exists() }
+                doLast {
+                    def classpathFiles = classpath.files
```

Comment:
❓ Why do we need to resolve this? We have this named variable now, but I don't see anything done with it.

Reply:
Right, this line is not needed, dropping it.